libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationView.kt

```diff
@@ -149,6 +146,13 @@ class DropInNavigationView @JvmOverloads constructor(
 
     override fun getLifecycle(): Lifecycle = viewLifecycleRegistry
 
+    override fun onKeyUp(keyCode: Int, event: KeyEvent): Boolean {
+        if (event.action == KeyEvent.ACTION_UP && keyCode == KeyEvent.KEYCODE_BACK) {
+            return viewModel.backPressManager.handleOnBackPressed()
+        }
+        return super.onKeyUp(keyCode, event)
+    }
+
```

Comment:
@kmadsen I tested your `BackStackComponent` approach but unfortunately its `OnKeyListener` is not being called. Can't figure out why. You can test it by commenting out this `onKeyUp` method and uncommenting the `BackPressCoordinator` above.

Luckily overriding `onKeyUp` here and delegating of back pressed handling directly to backPressManager does work.

What do you think of this approach?

Reply:
> I tested your BackStackComponent approach but unfortunately its OnKeyListener is not being called. Can't figure out why.

Interesting because I think onKeyUp is a similar mechanism 🤔 

It seems to be a solution for some people, which ViewGroup are you attaching to? https://stackoverflow.com/questions/33506693/handling-back-button-in-custom-view

This works for me on the main branch in `DropInNavigationView init`
``` kotlin
    init {
        isFocusableInTouchMode = true
        requestFocus()
        setOnKeyListener { v, keyCode, event ->
            if (event.action == KeyEvent.ACTION_UP && keyCode == KeyEvent.KEYCODE_BACK) {
                Log.i("kyle_debug", "onBack pressed")
            }
            false
        }
```

Reply:
Wondering if this is a hardware vs software keyboard issue 🤔

Reply:
> ... which ViewGroup are you attaching to?

DropInNavigationView